src/helpers/storage.js

```diff
@@ -0,0 +1,24 @@
+const set = (key, value) => {
+  window.localStorage.setItem(key, value)
+}
```

Comment:
You could turn these into one line functions like so:
```js
const set = (key, value) => window.localStorage.setItem(key, value)
```

Reply:
Hey @smakosh 👋 , prettier sometimes doesn't work in my vscode.
I will check that later tonight, and for sure I will add the ts version.